invenio_vocabularies/contrib/subjects/mappings/os-v1/subjects/subject-v1.0.0.json

```diff
@@ -1,5 +1,16 @@
 {
   "mappings": {
+    "dynamic_templates": [
+      {
+        "i18n_title": {
+          "path_match": "title.*",
+          "match_mapping_type": "string",
+          "mapping": {
+            "type": "search_as_you_type"
+          }
```

Comment:
minor/shelve: we should apply the same search optimizations that are available at https://github.com/inveniosoftware/invenio-vocabularies/pull/372. Can be done in a separate PR